models/iosxr/bgp_global/iosxr_bgp_global.yaml

```diff
@@ -0,0 +1,826 @@
+            cluster_id: *cluster_id
+            description: &desc
+              description: Neighbor specific description.
+              type: str
+            dmz_link_bandwidth: &dmz_link_bw
+              description: Propagate the DMZ link bandwidth.
+              type: dict
+              suboptions:
+                inheritance_disable:
+                  description: Do not inherit this configuration from parent group.
+                  type: bool
+            dscp: &dscp
+              description: Set IP DSCP (DiffServ CodePoint).Please refer vendor document for valid entries.
+              type: int
+            ebgp_multihop: &ebgp_multihop
+              description: Allow EBGP neighbors not on directly connected networks.
+              type: dict
+              suboptions:
+                value:
+                  description: maximum hop count.Example- <1-255>.
+                  type: int
+                mpls:
+                  description: Disable BGP MPLS forwarding.
+                  type: bool
+            ebgp_recv_extcommunity_dmz: &ebgp_recv_dmz
+              description: Receive extcommunity dmz link bandwidth from ebgp neighbor.
+              type: dict
+              suboptions:
+                inheritance_disable:
+                  description: Prevent ebgp-recv-community-dmz from being inherited from parent
+                  type: bool
+            ebgp_send_extcommunity_dmz: &ebgp_send_dmg
+              description: Send extcommunity dmz link bandwidth from ebgp neighbor.
+              type: dict
+              suboptions:
+                inheritance_disable:
+                  description: Prevent ebgp-send-community-dmz from being inherited from parent
+                  type: bool
+                cumulatie:
+                  description: Send cumulative community dmz link bandwidth of all multipaths to ebgp neighbor.
+                  type: bool
+            egress_engineering: &egress_eng
+              type: dict
+              description: Enable egress peer engineering for this neighbor.
+              suboptions:
+                inheritance_disable:
+                  description: Prevent egress_engineering from being inherited from parent
+                  type: bool
+            enforce_first_as: *enforce_first_as
+            graceful_maintenance: &graceful_maintenance
+              description:
+                Attributes for Graceful Maintenance. This will cause neighbors to de-prefer routes from this router and
+                choose alternates. This allows the router to be brought in or out of service gracefully.
+              type: dict
+              suboptions:
+                activate:
+                  description: Routes will be announced with the graceful maintenance attributes while activated either here or under router
+                    bgp configuration.
+                  type: dict
+                  suboptions:
+                    inheritance-disable:
```

Comment:
`-` is not a valid character in parameter name.

Reply:
Ack, made changes